docs/source/algorithms/EstimateDivergence-v1.rst

```diff
@@ -0,0 +1,52 @@
+
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm estimates the divergence of a diffraction instrument using equation 6.9 of Windsor
+
+.. math:: \Delta\theta_{div} = \frac{1}{2}
+          \sqrt{\Delta(2\theta)^2 + \alpha_0
+          + \frac{4\left(\beta_0^2 + \beta_1^2\right)}{\sin^2(2\theta)}}
+
+Where :math:`\Delta\theta_{div}` is the divergence, :math:`\Delta(2\theta)` is the angular uncertainty due to the detector size, :math:`\alpha_0` is the uncertainty in the incident collimation in the scattering plane, and the :math:`\beta` terms are the angular uncertainties out of the scattering plane.
+
```

Comment:
Not clear where you have two uncertainties out of the scattering plane

Reply:
Me neither. Windsor is really unclear about what the two angles actually are.